<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>572</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QComboBox" name="cbCanList_0">
              <property name="sizeAdjustPolicy">
               <enum>QComboBox::AdjustToMinimumContentsLengthWithIcon</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="bnInit_0">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>init</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QWidget" name="widgetParams_0" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>2</number>
             </property>
             <property name="topMargin">
              <number>2</number>
             </property>
             <property name="rightMargin">
              <number>2</number>
             </property>
             <property name="bottomMargin">
              <number>2</number>
             </property>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <property name="spacing">
                <number>0</number>
               </property>
               <item>
                <widget class="QCheckBox" name="checkErrorFilter_0">
                 <property name="minimumSize">
                  <size>
                   <width>90</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>90</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Error filter</string>
                 </property>
                 <property name="tristate">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="editErrorFilter_0"/>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QCheckBox" name="checkLoopback_0">
               <property name="text">
                <string>Loopback</string>
               </property>
               <property name="tristate">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="checkReceiveOwn_0">
               <property name="text">
                <string>Receive own</string>
               </property>
               <property name="tristate">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="checkCANFD_0">
               <property name="text">
                <string>CAN FD</string>
               </property>
               <property name="tristate">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_6">
               <property name="spacing">
                <number>0</number>
               </property>
               <item>
                <widget class="QCheckBox" name="checkBitrate_0">
                 <property name="minimumSize">
                  <size>
                   <width>90</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>90</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Bitrate</string>
                 </property>
                 <property name="tristate">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="editBitrate_0"/>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QCheckBox" name="checkExtendedFrame_0">
               <property name="text">
                <string>Extended frame</string>
               </property>
               <property name="tristate">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="checkFlexRateFormat_0">
               <property name="text">
                <string>Flex rate format</string>
               </property>
               <property name="tristate">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLineEdit" name="editWrite_0">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>112233445566778899001122334455667788990011223344556677889900112233445566778899</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="bnWrite_0">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>write</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QComboBox" name="cbCanList_1"/>
            </item>
            <item>
             <widget class="QPushButton" name="bnInit_1">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>init</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QWidget" name="widgetParams_1" native="true">
            <layout class="QVBoxLayout" name="verticalLayout">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>2</number>
             </property>
             <property name="topMargin">
              <number>2</number>
             </property>
             <property name="rightMargin">
              <number>2</number>
             </property>
             <property name="bottomMargin">
              <number>2</number>
             </property>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_7">
               <property name="spacing">
                <number>0</number>
               </property>
               <item>
                <widget class="QCheckBox" name="checkErrorFilter_1">
                 <property name="minimumSize">
                  <size>
                   <width>90</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>90</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Error filter</string>
                 </property>
                 <property name="tristate">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="editErrorFilter_1"/>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QCheckBox" name="checkLoopback_1">
               <property name="text">
                <string>Loopback</string>
               </property>
               <property name="tristate">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="checkReceiveOwn_1">
               <property name="text">
                <string>Receive own</string>
               </property>
               <property name="tristate">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="checkCANFD_1">
               <property name="text">
                <string>CAN FD</string>
               </property>
               <property name="tristate">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_8">
               <property name="spacing">
                <number>0</number>
               </property>
               <item>
                <widget class="QCheckBox" name="checkBitrate_1">
                 <property name="minimumSize">
                  <size>
                   <width>90</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>90</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Bitrate</string>
                 </property>
                 <property name="tristate">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="editBitrate_1"/>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QCheckBox" name="checkExtendedFrame_1">
               <property name="text">
                <string>Extended frame</string>
               </property>
               <property name="tristate">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="checkFlexRateFormat_1">
               <property name="text">
                <string>Flex rate format</string>
               </property>
               <property name="tristate">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLineEdit" name="editWrite_1">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>445566</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="bnWrite_1">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>write</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="ptextLog">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="plainText">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>572</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
